sources/packages/backend/apps/api/src/route-controllers/education-program-offering/_tests_/e2e/education-program-offering.institutions.controller.getOfferingSummary.e2e-spec.ts

```diff
@@ -0,0 +1,304 @@
+import { HttpStatus, INestApplication } from "@nestjs/common";
```

Comment:
My understanding is that we should add at least a simple test suite for all external controllers because they may have minor differences in things like parameters, security, etc. Which means adding an e2d test for `education-program-offering.aest.controller.ts`. But I do see in this case the code is almost identical. So let's discuss this with the team.

Reply:
@dheepak-aot whats your opinion here?

Reply:
I won't do this as part of this PR since it is already taking too much time/effort